Enable tls authentication for otlp grpc exporter (#635)
open-telemetry/opentelemetry-cpp: examples/otlp/README.md

calls a `foo_library` which has been instrumented using the [OpenTelemetry
API](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/api).

> To enable tls authentication for OTLP grpc exporter, SslCredentials can be used by

Reviewer: ```suggestion
To enable TLS authentication for OTLP grpc exporter, SslCredentials can be used by
```

> specifying the path to cacert pem file via OtlpExporterOptions. The path to such a

Reviewer 2: ```suggestion
specifying the path to client certificate pem file via OtlpExporterOptions. The path to such a
```

Reviewer 2: Correct me - but ideally this would be client certificate file - containing client cer, client key and CA certificate chain, In that case, we can just say `client certificate` instead of `cacert` ?

> specifying the path to client certificate pem file via OtlpExporterOptions. The path to such a

Reviewer 2: ```suggestion
specifying the path to client certificate pem file, or the string containing this certificate via OtlpExporterOptions. The path to such a
```

Reviewer 2: Sorry for one last comment :)

Author: no worries, makes sense, forgot to update.
